source/2019-12-30-countdown-to-the-new-year-2019-ember-exam.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Countdown to The New Year: Ember Exam
+author: Jessica Jordan
+tags: 2019, Recent Posts, Testing
```

Comment:
let's take out Recent Posts, so it doesn't bury the 3.15 release blog post- I wonder if we can have a different tag here- `DecEmber` maybe?

Reply:
Yes, that sounds great! I also used the tags from your PR #365 as a reference :+1: